project/annotations/management/commands/replace_label_in_source.py

```diff
@@ -102,5 +102,5 @@ def handle(self, *args, **options):
         source.labelset.locallabel_set.get(global_label=old_label).delete()
 
         # Reset the backend state for this source.
-        backend_tasks.reset_after_labelset_change.apply_async(
+        backend_tasks.reset_backend_for_source.apply_async(
```

Comment:
This is actually not correct. `reset_after_labelset_change ` doesn't reset the feature extractions, only the classifications. So you need to maybe keep the old one and then create a new one `reset_backend_for_source` that calls `reset_after_labelset_change` but also `reset_features` for all images in the source.

Reply:
... obviously, rename the function however you like. The old names are a bit weird.

Reply:
Ah, thanks for the catch. Yeah, I definitely misread what that task was doing. I'll fix it later today.

Reply:
Added a commit for that. Took longer than expected because I wanted to improve some related unit tests.

Reply:
Tnx. Let me try it out.

Reply:
I just added 1 more commit. The reset-task tests now check that the backend objects get re-created after being deleted.